src/qml/Mysthea/Pages/Lore.qml

```diff
@@ -1,28 +1,40 @@
-import QtQuick 2.0
+    import QtQuick 2.0
 import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.5
 
 import Mysthea.Theme 1.0
 
 Page {
     id: root
     title: qsTr("LORE")
-
     objectName: PageName.lorePage
+    topPadding: 0
+    padding: 20
```

Comment:
`bottomPadding` should be 0 too, otherwise text and images are cut before the end of the screen.
Handle it correctly.